test/Jacobian_test.cpp

```diff
@@ -32,6 +32,10 @@
 #include <numeric>
 #include <vector>
 
+#ifndef M_PI
+#define M_PI 3.141592653589793238462643383279502884197169399
+#endif
```

Comment:
Please see this: https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170  - basically define `_USE_MATH_DEFINES` at the top of the file and the constant will be available when including `<cmath>`